db/migrate/20230222101705_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, index:true
+      t.references :post, null: false, foreign_key: true, index:true
+      t.string :text
```

Comment:
- [ ] The text needs to be t.text which allows more characters(words) to be included